modules/packages/LinearAlgebra.chpl

```diff
@@ -702,6 +702,30 @@ proc _matmatMult(A: [?Adom] ?eltType, B: [?Bdom] eltType)
   return C;
 }
 
+/*
+  Returns the inverse of ``A`` square matrix A.
```

Comment:
Not necessary in this PR, but I think `matPow(A, -1)` should eventually return this. It currently halts for negative values.